fontc/src/main.rs

```diff
@@ -253,7 +258,8 @@ fn main() -> Result<(), Error> {
     let paths = Paths::new(Path::new("build"));
     let build_dir = require_dir(paths.build_dir())?;
     require_dir(paths.glyph_ir_dir())?;
-    let (config, prev_inputs) = init(&build_dir, Args::parse()).map_err(Error::IoError)?;
+    let config = Config::new(Args::parse(), build_dir)?;
```

Comment:
It's not ideal that we hardcode our build directory to be "build/" in the current directory, maybe it should become part of Args... then you would only need the args to initialize a Config. 
feel free to do that another time.

Reply:
https://github.com/googlefonts/fontmake-rs/issues/53

Reply:
Good catch, will fix. +1 to standalone issue.